### ISSUE TITLE
Refactor: Dice Page +/- Buttons

### ISSUE THREAD

**Issue by LKessell** [SWRPG-Capstone/swrpg-fe#70]
The increase/decrease dice number buttons on the dice page are currently created using `div` elements. For better accessibility, these should be converted to using the `button` element instead. Will likely need to tweak CSS
